e2e/test/scenarios/admin/i18n/content-translation/dashboard-tab-names.cy.spec.ts

```diff
@@ -0,0 +1,33 @@
+import {
+  NORMAL_USER_ID,
+  ORDERS_DASHBOARD_ID,
+} from "e2e/support/cypress_sample_instance_data";
+import type { DictionaryArray } from "metabase/i18n/types";
+
+import { uploadTranslationDictionary } from "./helpers/e2e-content-translation-helpers";
+
+const { H } = cy;
+
+describe("scenarios > content translation > dashboard tab names", () => {
+  const textCardTranslations: DictionaryArray = [
+    { locale: "de", msgid: "Tab 1", msgstr: "Reiter 1" },
+  ];
+
+  beforeEach(() => {
+    cy.intercept("POST", "api/ee/content-translation/upload-dictionary").as(
+      "uploadDictionary",
+    );
+    H.restore();
+    cy.signInAsAdmin();
+    H.setTokenFeatures("all");
+    uploadTranslationDictionary(textCardTranslations);
+    cy.request("PUT", `/api/user/${NORMAL_USER_ID}`, { locale: "de" });
```

Comment:
I'm a little surprised you don't need to apply the translation to the `ADMIN_USER_ID` for this test to work